pymc_experimental/tests/distributions/test_continuous.py

```diff
@@ -149,3 +150,54 @@ class TestGenExtreme(BaseTestDistributionRandom):
         "check_pymc_draws_match_reference",
         "check_rv_size",
     ]
+
+
+class TestChiClass:
+    """
+    Wrapper class so that tests of experimental additions can be dropped into
+    PyMC directly on adoption.
+    """
+
+    def test_logp(self):
+        check_logp(
+            Chi,
+            Rplus,
+            {"nu": Rplus},
+            lambda value, nu: sp.chi.logpdf(value, df=nu),
+        )
+
+    def test_logcdf(self):
+        check_logcdf(
+            Chi,
+            Rplus,
+            {"nu": Rplus},
+            lambda value, nu: sp.chi.logcdf(value, df=nu),
+        )
+
+    """
+    @pytest.mark.parametrize(
+        "nu, size, expected",
+        [
+            (1, None, 1),
+            (1, 5, np.full(5, 1)),
+            (np.arange(1, 6), None, np.arange(1, 6)),
+        ],
+    )
+    def test_chi_moment(self, nu, size, expected):
+        with pm.Model() as model:
+            Chi("x", nu=nu, size=size)
+        assert_moment_is_expected(model, expected)
```

Comment:
No need for this. We are adding default moments to CustomDist in https://github.com/pymc-devs/pymc/pull/6873